build.sbt

```diff
@@ -65,7 +66,10 @@ lazy val coreDependencies = Seq(
   "com.typesafe.akka" %% "akka-remote" % versions.akka,
   "ch.megard" %% "akka-http-cors" % versions.akkaHttpCors,
   "de.heikoseeberger" %% "akka-http-json4s" % "1.16.1",
-  "org.json4s" %% "json4s-native" % "3.6.3",
+  "org.json4s" %% "json4s-native" % "3.6.2",
+  "org.json4s" %% "json4s-ext" % "3.6.2",
+  "org.json4s" %% "json4s-jackson" % "3.6.2",
```

Comment:
this seems weird -- why do we need jackson and native?

Reply:
Just came from example for schema validator lib, can change though

Reply:
Oh wait I think this is a jackson specific thing. I'm not sure it can be done with json4s